live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -361,3 +361,12 @@
     path: /etc/default/grub.d/kdump-tools.cfg
     regexp: '^GRUB_CMDLINE_LINUX_DEFAULT='
     line: 'GRUB_CMDLINE_LINUX_DEFAULT="$GRUB_CMDLINE_LINUX_DEFAULT crashkernel=1024M-:512M"'
+
+# Panic on ZFS assertion failures
+- lineinfile:
+    path: /etc/modprobe.d/zfs.conf
+    create: yes
+    line: 'options spl spl_panic_halt=1'
+# Because the spl module is loaded from the initramfs when using zfs on root,
+# we need to rebuild the initramfs to account for the modprobe.d change.
+- shell: update-initramfs -u
```

Comment:
It'd be better to run this command as an Ansible "handler", such that it's only executed when the "zfs.conf" file changes. We don't have an example of a handler yet in this repository, but it's pretty simple; I can give you some help if you want to look into doing it that way.

With that said, if you'd rather just land this as is, I'm OK with that. This isn't quite as idempotent as it could be, but given our use case, that's not a practical concern.